pallets/assets/src/lib.rs

```diff
@@ -1043,4 +1043,11 @@ impl<T: Config>
         }
         r
     }
+
+    fn get_asset_owner(asset_id: &T::AssetId) -> Result<T::AccountId, DispatchError> {
+        let owner = Self::asset_owner(asset_id)
+            .ok_or(Error::<T>::AssetIdNotExists)
+            .unwrap();
```

Comment:
`unwrap` leads to panic. It is necessary to return the error above, not panic.
Use `unwrap` only in tests please.
```suggestion
        let owner = Self::asset_owner(asset_id) .ok_or(Error::<T>::AssetIdNotExists)?;
```